### PR TITLE
[IMPL] Misc. Improvements

### PR DESCRIPTION
**Pull request by HausemasterIssue**
just improved some misc shit, sorry for not putting it all in one pr i didnt think i would do this but here i am

also linus, i figure out that silent swap on strict servers, you want to make sure your hands arent doing anything (e.g eating, placing crystals) while silent swap is swapping ur shit, i think you could do that with the mc.player.stopActiveHand() method. also another layer of shit needed is to instantly swap to the obi or something, then wait 1.5 ticks. you should wait 1.5 ticks because i saw soulbond say for silent swap on konas you want to have a 1.5 tick swap delay so im guessing you need a 1.5 tick delay between swap packets so ye. i was thinking of adding it but it might break shit so ill test it by myself. 

basically for silent swap, you want to make sure ur hands arent doing anything, and you want to wait 1.5 ticks between swap packets for 2b.

### CODE REVIEW

**Comment by Sxmurai** [comment]
you dont need to be a master coder to make a client, you need to know the basics and "best practices"

File: src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java [hunk 1]
```diff
@@ -107,7 +107,7 @@ public void onPacketSend(PacketEvent.PacketSendEvent event) {
 
     @SubscribeEvent
     public void onInputUpdate(InputUpdateEvent event) {
-        if (this.isSlowed()) {
+        if (this.isSlowed() && !ncpStrict.getValue()) {
```
[the remark this answers is not in the document]
Comment (Sxmurai) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java:110]: yeah but the check you do here makes 0 sense. until this is removed, its not gonna be merged cause it completely removes the clientsided noslow effect, which is the entire point of this module.

[the remark this answers is not in the document]
Comment (Sxmurai) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java:110]: and why dont you always not wanna do it? it legit is what makes you not be slowed down by items...

[the remark this answers is not in the document]
Comment (HausemasterIssue) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java:110]: what im trying to do is make it so noslow works normally if ur strafing in the air, but add an option where if your on the ground just walking around with noslow it wont do shit and just slow you down normally. 

perhaps we might need that code for noslow to work on the ground or while strafing.

[the remark this answers is not in the document]
Comment (Sxmurai) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java:110]: this isnt the right option though???? the fuck???? do you even read the descriptions of the settings and the code?

[the remark this answers is not in the document]
Comment (HausemasterIssue) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java:110]: the digging packet is what allows u to semi-bypass it but if you move like 0.5 blocks u start getting rubberbanded right? so shoulnd't you have a way to choose if u wanna get rubberbanded if u walk 0.5 blocks while eating, or to go slower if your on the ground but not get rubberbanded?

[the remark this answers is not in the document]
Comment (Sxmurai) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java:110]: what are you even talking about? that packet has nothing to do with rubberbanding on strict servers? you still have to multiply the strafe and forward values by 5.0f.

this is because Minecraft's code multiplies those two values by 0.2f when a hand is active to slow you down, and we reverse that with 5.0f.

regardless, you always want this. your thing here makes no sense.

[the remark this answers is not in the document]
Comment (Sxmurai) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java:110]: and the sneak bypass only works while strafing, nobody has figured out a way to bypass it without it... again, im not merging this pr until you fix this because this will entirely fuck up NoSlow on servers that dont require you to have a stricter NoSlow config.

[the remark this answers is not in the document]
Comment (Sxmurai) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java:110]: im not 100% sure what you aren't understanding, you adding this check makes 0 sense.

[the remark this answers is not in the document]
Comment (Sxmurai) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java:110]: and i understand what youre trying to say, but your implementation is fucking horrible.

a better way to do that is to check if the player is on the ground, and then not to do that if the sneak bypass is enabled, then it wouldn't rubberband you.

but doing this with no onground check and no thought behind it will hinder the use of the module, making it useless.

[the remark this answers is not in the document]
Comment (HausemasterIssue) [reply src/main/java/cope/cosmos/client/features/modules/movement/NoSlow.java]: done in latest commit

**Comment by HausemasterIssue** [comment]
yo wtf
